### PR TITLE
add warning to upload clean data without duplicates or missing abstracts

### PR DESCRIPTION
**Pull request by Rensvandeschoot**

### CODE REVIEW

File: docs/source/features/pre_screening.rst [hunk 1]
```diff
@@ -34,6 +34,15 @@ To select a dataset:
 .. figure:: ../../images/asreview_prescreening_datasets.png
    :alt: ASReview dataset selector
 
+.. warning::
+
+    If you upload your own data make sure to remove duplicates and make sure
```
Comment (SagevdBrand) [inline docs/source/features/pre_screening.rst:39]: Perhaps this sentence can be shortened to:
"If you upload your own data, make sure to remove duplicates and to retrieve as many abstracts as possible (`don't know how (..))."

**Review by SagevdBrand — COMMENTED** [review]
Nice! Thanks for highlighting this for the users! I only have a small comment for the first sentence 👍

File: docs/source/lab/simulation.rst [hunk 2]
```diff
@@ -11,3 +11,12 @@ See the following resources for  information on running a simulation:
...
 - :doc:`../guides/sim_overview`
+
+
+.. warning::
+
+    If you upload your own data, make sure to remove duplicates and to retrieve 
+    as many abstracts as possible (`don't know how?
+    <https://asreview.nl/the-importance-of-abstracts/>`_). With clean data you
+    benefit most from what :doc:`active learning <../guides/activelearning>` 
+    has to offer you.
```
Comment (GerbrichFerdinands) [inline docs/source/lab/simulation.rst:22]: Maybe this is not the right place, as the ideal situation, the cleaning has happened before you run a simulation. Then, a simulation shows your process as is. On the other hand, I see the value of mentioning the GIGO principle here.

**Review by GerbrichFerdinands — CHANGES_REQUESTED** [review]
Nice, great addition to the docs. These changes give two warnings at the moment that need to be fixed.

**Comment by J535D165** [comment]
Thanks @Rensvandeschoot @GerbrichFerdinands and @SagevdBrand!

**Pull request merged**